<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>edit_of_task</class>
 <widget class="QWidget" name="edit_of_task">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1031</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="edittopic_groupbox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>211</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>edit the Topic of this task</string>
   </property>
   <widget class="QLabel" name="topic_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Topic:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edittopic_lineedit">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>60</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="editdescription_groupbox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>230</y>
     <width>261</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>edit the Description of this task</string>
   </property>
   <widget class="QLabel" name="description_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Description:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="editdescription_textedit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>201</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="editpriority_groupbox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>301</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>edit the Priority of this task</string>
   </property>
   <widget class="QLabel" name="editpriority_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>edit the Priority of this task</string>
    </property>
   </widget>
   <widget class="QComboBox" name="editpriority_combobox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="edittimeanddate_groupbox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>681</width>
     <height>431</height>
    </rect>
   </property>
   <property name="title">
    <string>Edit Date and Time of this task</string>
   </property>
   <widget class="QLabel" name="edittime_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit the time of your topic in hh:mm:ss mode:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edittime_lineedit">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="editdate_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>311</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit the day of this task :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="show_calender_pushButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>160</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>show the calender</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="allowedit_pushbutton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>490</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Quitedit_pushbutton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>490</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>which topic do you want to edit?</string>
   </property>
   <widget class="QLineEdit" name="whichtopic_lineEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Quitedit_pushbutton</sender>
   <signal>clicked()</signal>
   <receiver>edit_of_task</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>609</x>
     <y>506</y>
    </hint>
    <hint type="destinationlabel">
     <x>751</x>
     <y>544</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
